app/styles/alert.less

```diff
@@ -13,6 +13,10 @@
     padding: @spacing-xx-sm;
     gap: @spacing-xx-sm;
     background-color: var(--color-gray-50);
+
+    &--plain {
+      background-color: var(--color-white);
```

Comment:
There is an error in the tech spec, the right color when plain is:
- true -> background-color: var(--color-gray-50);
- false -> background-color: var(--color-white);

I check on design & Victor to be sure